CIS560-RecipeManager/CIS560-RecipeManager/Pantry/Pantry.cs

```diff
@@ -0,0 +1,44 @@
+﻿/**
+ * Pantry.cs
+ * Class representing a pantry, which contains a list of 
+ * ingredients that the are currently in the pantry. 
+ * Authors: CIS560 Project Team 1
+ */
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CIS560_RecipeManager
+{
+    public class Pantry
```

Comment:
I like the idea of adding a Pantry class.  I think it makes more sense than our current design, which is each ingredient object holding a ```PantryQuantity``` field that holds its quantity in the ```Pantry```.  I think we should keep the Pantry class and remove the ```PantryQuantity``` field from ```Ingredient```.

Reply:
We would just want one instance of the Pantry class, as a user only has one Pantry.  We could have the HomeController create this instance and pass it as a parameter to the constructors of the PantryController, RecipeController, and ShoppingListController.